### PR TITLE
PR#1

### PR DESCRIPTION
**Pull request by VTerenya**

### CODE REVIEW

File: wordCount.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	ss := strings.Split(s, " ")
+	fmt.Println(ss)
+	visited := make(map[string]bool)
+	result := make(map[string]int)
+	for _, item := range ss {
+		if visited[item] == true || item==""{
+			continue
+		} else {
+			visited[item] = true
+			c:=countSubString(item,ss)
```
Comment (rtemb) [inline wordCount.go]: It's not effective in terms of performance to run for cycle inside the for cycle. You don't have to use nesting cycles to implement word count logic.

File: wordCount.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+func wordCount(s string) map[string]int {
+	ss := strings.Split(s, " ")
```
Comment (rtemb) [inline wordCount.go]: `ss ` bad name for variable. Please use something more meaningful

File: wordCount.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+func countSubString(s string, ss []string) int {
+	k:=0
+	for _,item := range ss{
+		if item == s{
+			k++
+		}
+	}
+	return k
```
Comment (rtemb) [inline wordCount.go]: Name of the following vars `k`, `ss`, `s` also need to be changed.

File: wordCount.go [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+func main() {
+	m := wordCount(" as sa as fe we e e e")
+	fmt.Println(m)
```
Comment (rtemb) [inline wordCount.go]: Please change add supporting to run your app from the CLI with passing any words there. For example: 
`wordcount "apple, orange, lemon, lemon"`

**Comment by rtemb** [comment]
Please, add also some short sensible title to your pr.

**Comment by VTerenya** [comment]
Thank you for your review, You can check my solution again. I fixed all issues.

**Review by rtemb — APPROVED** [review]

**Pull request closed without merging**